container/shared/src/main/filtered-resources/etc/org.apache.karaf.features.cfg

```diff
@@ -0,0 +1,93 @@
+################################################################################
+#
+#    Licensed to the Apache Software Foundation (ASF) under one or more
+#    contributor license agreements.  See the NOTICE file distributed with
+#    this work for additional information regarding copyright ownership.
+#    The ASF licenses this file to You under the Apache License, Version 2.0
+#    (the "License"); you may not use this file except in compliance with
+#    the License.  You may obtain a copy of the License at
+#
+#       http://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS,
+#    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#    See the License for the specific language governing permissions and
+#    limitations under the License.
+#
+################################################################################
+
+#
+# Comma separated list of features repositories to register by default
+#
+featuresRepositories = \
+    mvn:org.apache.karaf.features/framework/${karafVersion}/xml/features, \
+    mvn:org.apache.karaf.features/spring/${karafVersion}/xml/features, \
+    mvn:org.opennms.karaf/opennms/${project.version}/xml/karaf-extensions, \
+    mvn:org.apache.karaf.features/standard/${karafVersion}/xml/features, \
+    mvn:org.apache.karaf.features/spring-legacy/${karafVersion}/xml/features, \
+    mvn:io.hawt/hawtio-karaf/2.0.0/xml/features
+
+#
+# Comma separated list of features to install at startup
+#
+featuresBoot = \
```

Comment:
Define this file so that we can control the order of the features listed in `featuresBoot`. Don't rely on the file generated by the `karaf-maven-plugin` in https://github.com/OpenNMS/opennms/blob/develop/container/shared/pom.xml#L70 anymore.